x-pack/docs/en/rest-api/security/saml-authenticate-api.asciidoc

```diff
@@ -0,0 +1,86 @@
+[role="xpack"]
+[[security-api-saml-authenticate]]
+=== SAML authenticate API
+
+Submits a SAML response message to {es} for consumption.
+
+[[security-api-saml-authenticate-request]]
+==== {api-request-title}
+
+`POST /_security/saml/authenticate`
+
+[[security-api-saml-authenticate-desc]]
+==== {api-description-title}
+
+The SAML response can be a response to a SAML authentication request that was
+previously created using the
+<<security-api-saml-prepare-authentication, SAML prepare authentication API>>.
+It can also be an unsolicited SAML response in the case of an IdP-initiated
+single sign-on (SSO) flow. After successful validation, {es} responds with an
+{es} internal access token and refresh token that can be subsequently used for 
+authentication. This API endpoint basically exchanges SAML responses that
+indicate successful authentication in the IdP for {es} access and refresh tokens,
+which can be used for authentication against {es}.
+
+{es} exposes all the necessary SAML related functionality via the SAML APIs.
+These APIs are used internally by {kib} in order to provide SAML based
+authentication, but can also be used by other, custom web applications or other
+clients. See also
+<<security-api-saml-prepare-authentication,SAML prepare authentication API>>,
+<<security-api-saml-invalidate,SAML invalidate API>> and
+<<security-api-saml-logout,SAML logout API>>.
+
+
+[[security-api-saml-authenticate-request-body]]
+==== {api-request-body-title}
+
+`content`::
+  (string) The SAML response as it was sent by the user's browser, usually a
```

Comment:
We should also indicate whether this property is Required or Optional, for example:

```suggestion
  (Required, string) The SAML response as it was sent by the user's browser, usually a
```